src/components/designer/taro/actions/NewAddressModal.tsx

```diff
@@ -0,0 +1,185 @@
+    setSelectedBalance(value);
+    setSelectedGenesisBootstrapInfo(value.genesisBootstrapInfo);
+    form.setFieldsValue({
+      genesisBootstrapInfo: value.genesisBootstrapInfo,
+      amount: value.type == PTARO.TARO_ASSET_TYPE.COLLECTIBLE ? 1 : 10,
+    });
+  };
+
+  //submit
+  const newAddressAsync = useAsyncCallback(async (payload: NewAddressPayload) => {
+    try {
+      const res = await getNewAddress(payload);
+      setTaroAddress(res.encoded);
+    } catch (error: any) {
+      notify({ message: l('submitError'), error });
+    }
+  });
+
+  const handleSubmit = (values: { genesisBootstrapInfo: string; amount: string }) => {
+    if (selectedGenesisBootstrapInfo) {
+      const payload: NewAddressPayload = {
+        node: thisTaroNode,
+        genesisBootstrapInfo: values.genesisBootstrapInfo,
+        amount: values.amount,
+      };
+      newAddressAsync.execute(payload);
+    }
+  };
+
+  const handleCopy = () => {
+    message.success(l('copied', { address: ellipseInner(taroAddress, 10, 10) }), 2);
+    hideNewAddress();
+  };
+
+  let cmp: ReactNode;
+  if (!taroAddress) {
+    cmp = (
+      <>
+        <Form
+          form={form}
+          layout="vertical"
+          colon={false}
+          initialValues={{
+            amount: '10',
+            node: '',
+            genesisBootstrapInfo: '',
+          }}
+          onFinish={handleSubmit}
+        >
+          <Form.Item name="genesisBootstrapInfo" label={l('genesisBootstrapInfo')}>
+            <Input.TextArea
+              rows={5}
+              placeholder={l('genesisBootstrapInfo')}
+              onChange={e => setSelectedGenesisBootstrapInfo(e.target.value)}
+            />
+          </Form.Item>
+          <Styled.Spacer />
+
+          <Styled.TaroDataSelect
+            name="node"
+            label={l('selectBalance')}
+            taroNetworkNodes={otherTaroNodes}
+            selectBalances
+            onChange={v => handleSelectedBalance(v?.valueOf() as PTARO.TaroBalance)}
+          />
+          <Styled.Spacer />
+          <Form.Item label={l('amount')} name="amount">
+            <InputNumber
+              onChange={v => setSelectedAmount(v?.valueOf()?.toString() as string)}
+              min={'1'}
+              disabled={selectedBalance?.type === PTARO.TARO_ASSET_TYPE.COLLECTIBLE}
+            />
+          </Form.Item>
+        </Form>
+      </>
+    );
+  } else {
+    cmp = (
+      <Result
+        status="success"
+        title={l('successTitle')}
+        subTitle={l('successDesc', {
+          assetName: selectedBalance?.name,
+          amount: format(`${selectedAmount}`),
+        })}
+        extra={
+          <Form>
+            <Form.Item>
+              <CopyableInput label="Invoice" value={taroAddress} />
+            </Form.Item>
+            <Form.Item>
+              <CopyToClipboard text={taroAddress} onCopy={handleCopy}>
+                <Button type="primary">{l('copyClose')}</Button>
```

Comment:
THis should be "Address" instead of "Invoice".

Reply:
Done

Reply:
Done